### PR TITLE
feat: add open forms component stories

### PR DESCRIPTION
**Pull request by bddjong**
Adds open forms components to storybook.

**Components to add**
- [x] BSN field
- [x] Checkbox
- [x] Columns
- [x] Content
- [x] Cosign
- [x] Currency
- [x] DateTimeField
- [x] Datefield
- [x] TextField
- [x] EditGrid (Repeating group)
- [x] Email
- [x] FieldSet
- [x] IBANField
- [x] LicensePlateField
- [x] Number
- [x] PhoneNumberField
- [x] Radio
- [x] Select
- [x] Selectboxes
- [x] Signature
- [x] Textarea
- [x] TimeField

### CODE REVIEW

**Comment by bddjong** [comment]
Theres a few components that require a bit more digging or configuration. For example the `map` component. As well as Open Forms layout react components such as the progress indicators, buttons and toolbars.

File: packages/storybook/package.json [hunk 1]
```diff
@@ -22,6 +22,9 @@
     "storybook": "storybook dev --config-dir config/ --port 6006",
     "typecheck": "tsc --noEmit"
   },
+  "peerDependencies": {
+    "react-formio": "^4.3.0"
+  },
```
[the remark this answers is not in the document]
Comment (bddjong) [reply packages/storybook/package.json]: Deze hoort hier inderdaad niet!

**Comment by bddjong** [comment]
Refactored to move the FormIO configuration into a hook for the `RenderFormioForm` component

**Review by Robbert — APPROVED** [review]

File: packages/storybook/src/open-forms/BSN.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+import { SingleFormioComponent } from './StoryUtil';
+
+const meta: Meta = {
+  id: 'open-forms-bsn',
+  title: 'Open Formulieren/BSNField',
+  component: SingleFormioComponent,
+  args: {
+    key: 'bsn',
+    type: 'bsn',
+    label: 'Burgerservicenummer (BNS)',
```
Comment (YourivHDenHaag) [inline packages/storybook/src/open-forms/BSN.stories.tsx:12]: Kleine typo

**Review by YourivHDenHaag — APPROVED** [review]
Ziet er goed uit, thanks!

**Pull request merged**